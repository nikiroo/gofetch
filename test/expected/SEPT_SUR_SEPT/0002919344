                     GRÈVE ANNULÉE CHEZ EUROSTAR                   

  [Monde] 26-09-2018 18:00

  o News link: http://s.7s7.be/3480016
  o Source link: 


    Par: rédaction
    
    26/09/18 - 18h00 Source: Belga    © photo news.
    
    La  grève  prévue  au sein du personnel d'Eurostar à Londres le
    30  septembre  et  le  1er  octobre n'aura finalement pas lieu,
    a-t-on   appris   mercredi  auprès  du  service  de  presse  de
    l'opérateur ferroviaire. Le service se déroulera normalement.
    
    Le  personnel  Eurostar travaillant à la gare londonienne de St
    Pancras  avait appelé à une grève de 48 heures les 30 septembre
    et  1er octobre pour protester contre des conditions de travail
    "dangereuses".
    
    Le  syndicat RMT (Rail, Maritime and Transport) avait décidé de
    cette  grève,  en dénonçant plusieurs problèmes cet été qui ont
    conduit  les employés Eurostar de la gare à affronter la colère
    de voyageurs excédés.



