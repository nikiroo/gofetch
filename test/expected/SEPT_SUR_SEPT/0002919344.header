0Grève annulée chez Eurostar	null/SEPT_SUR_SEPT/0002919344		70
i  [Monde] 26-09-2018 18:00
i
i    La  grève  prévue  au sein du personnel d'Eurostar à Londres le
i    30  septembre  et  le  1er  octobre n'aura finalement pas lieu,
i    a-t-on appris ...
i
