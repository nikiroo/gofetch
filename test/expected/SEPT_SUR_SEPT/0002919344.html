<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Grève annulée chez Eurostar</h2>
	<div class='details'>([Monde] 26-09-2018 18:00)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0002919344</a></li>
        <li>News link: <a href='http://s.7s7.be/3480016'>http://s.7s7.be/3480016</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 18h00 Source: Belga    © photo news.<br/><br/>La grève prévue au sein du personnel d&#x27;Eurostar à Londres le 30 septembre et le 1er octobre n&#x27;aura finalement pas lieu, a-t-on appris mercredi auprès du service de presse de l&#x27;opérateur ferroviaire. Le service se déroulera normalement.<br/><br/>Le personnel Eurostar travaillant à la gare londonienne de St Pancras avait appelé à une grève de 48 heures les 30 septembre et 1er octobre pour protester contre des conditions de travail &quot;dangereuses&quot;.<br/><br/>Le syndicat RMT (Rail, Maritime and Transport) avait décidé de cette grève, en dénonçant plusieurs problèmes cet été qui ont conduit les employés Eurostar de la gare à affronter la colère de voyageurs excédés.
	</div>
<hr/>
</div>
</body>
